src/analysis/normed_space/operator_norm.lean

```diff
@@ -353,6 +353,70 @@ begin
       by rwa [mul_one] }
 end
 
+section uniformly_extend
+
+variables [complete_space F] {e : E →L[𝕜] G}
+
+section
+variables (h_e : uniform_inducing e) (h_dense : dense_range e)
+
+/-- Extension of a continuous linear map `f : E →L[𝕜] F`, with `E` a normed space and `F` a complete
+    normed space, along a uniform and dense embedding `e : E →L[𝕜] G`.  -/
+def extend : G →L[𝕜] F :=
+/- extension of `f` is continuous -/
+have cont : _ := (uniform_continuous_uniformly_extend h_e h_dense f.uniform_continuous).continuous,
+/- extension of `f` agrees with `f` on the domain of the embedding `e` -/
+have eq : _ := uniformly_extend_of_ind h_e h_dense f.uniform_continuous,
+{ to_fun := (h_e.dense_inducing h_dense).extend f,
+  add :=
+  begin
+    refine is_closed_property2 h_dense (is_closed_eq _ _) _,
+      { exact cont.comp (_root_.continuous_add continuous_fst continuous_snd) },
+      { exact _root_.continuous_add (cont.comp continuous_fst) (cont.comp continuous_snd) },
+      { assume x y, rw ← e.map_add, simp only [eq], exact f.map_add _ _  },
+  end,
+  smul := λk,
+  begin
+    refine is_closed_property h_dense (is_closed_eq _ _) _,
+      { exact cont.comp (continuous_smul continuous_const continuous_id)  },
+      { exact (continuous_smul continuous_const continuous_id).comp cont },
+      { assume x, rw ← map_smul, simp only [eq], exact map_smul _ _ _  },
+  end,
+  cont := cont
+}
+
+end
+
+section
+variables {M N : ℝ} (N0 : 0 ≤ N) (h_dense : dense_range e) (he : ∀x, ∥x∥ ≤ N * ∥e x∥)
+include N0
+
+local notation `ψ` := f.extend (uniform_embedding_of_bound _ _ he).to_uniform_inducing h_dense
+
+/-- If a dense embedding `e : E →L[𝕜] G` expands the norm by a constant factor `N`, and the norm of
+    `f : E →L[𝕜] G` is bounded by `M`, then the norm of the extension of `f` along `e` is bounded
+    by `M * N`. -/
+lemma op_norm_uniformly_extend_le (hf : ∥f∥ ≤ M): ∥ψ∥ ≤ M * N :=
```

Comment:
```suggestion
lemma op_norm_uniformly_extend_le (hf : ∥f∥ ≤ M) : ∥ψ∥ ≤ M * N :=
```

Reply:
Why don't you omit `M` completely and just write `∥ψ∥ ≤ N * ∥f∥`? If someone wants to use the lemma with an upper bound on `∥f∥`, he can just apply `le_trans`. This has also the advantage that you can then omit the assumption `0 ≤ N`: if `N` is negative, this can only happen if the space is `0`-dimensional, but then all norms are zero and the conclusion still holds (this is a minor point: if removing the assumption makes the proof too painful, you can keep it).